src/hobbits-widgets/spinbox64.h

```diff
@@ -0,0 +1,16 @@
+#ifndef SPINBOX64_H
+#define SPINBOX64_H
+
+#include <QWidget>
+
+class spinbox64 : public QWidget
```

Comment:
class name should be SpinBox64 - but more importantly, this shouldn't be in the pull request if it is incomplete